src/main/resources/wayf_widget_template.js

```diff
@@ -17,38 +17,26 @@
 const WAYF_BASE_URL = "https://wayf-cloud-sandbox.literatumonline.com";
 
 const AUTHORIZATION_HEADER_KEY = "Authorization";
-const AUTHORIZATION_HEADER_VALUE = "Bearer @PUBLISHER_JWT@";
+const AUTHORIZATION_HEADER_VALUE = "@PUBLISHER_JWT@";
 
-const GLOBAL_ID_HEADER_NAME = "X-Device-Id";
```

Comment:
yes. that's right. X-Device-Id is now the 'cookie name'. It is set by the server in the cookie and the browser sends this cookie back to the server in subsequent requests. This is completely transparent to the client. if you change the cookie name at the server the client is not affected.